src/framework/piral-core/src/helpers.tsx

```diff
@@ -118,7 +118,7 @@ export function createPiletOptions({
     });
   }
 
-  if (process.env.DEBUG_PILET !== undefined) {
+  if (window['dbg:pilet-api'] !== undefined) {
```

Comment:
Thinking here...

My desired behavior is that *always* bundle that block into the emulator (but *never* in the real app). Right now, this would not be the case...

So what we need is something like `process.env.DEBUG_PIRAL`, but only for emulator purposes. Maybe use `process.env.DEBUG_PILET` again - but really only to be `"on"` (i.e., something truthy) or `undefined`.

Reply:
yeah that makes sense